bin/mycroft-start

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Set the execution flag on this file

Reply:
Whoops, Windows stripped the X bit.  Will do.